keyboards/kprepublic/jj50/keymaps/via/keymap.c

```diff
@@ -0,0 +1,142 @@
+     * |------+------+------+------+------+------|------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      |      |      |      |      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |             |      |      |      |      |      |
+     * `-----------------------------------------------------------------------------------'
+     */
+    [_ADJUST] = LAYOUT(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+        _______, RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL,
+        KC_CAPS, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    ),
+
+    /* Fn (rgblight and backlight layout)
+     * ,-----------------------------------------------------------------------------------.
+     * |      |      |      |      |      |      |      |      |      |      |      |      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      |      |RGBVA-|RGBVA+|      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      |      |RGBSA-|RGBSA+|      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      |RGBTOG|RGBHU-|RGBHU+|RGBMOD|
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |             |      |BLTOGG|  BL- |  BL+ |BLBRTG|
+     * `-----------------------------------------------------------------------------------'
+     */
+    [_FN] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_VAD, RGB_VAI, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_SAD, RGB_SAI, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, RGB_TOG, RGB_HUD, RGB_HUI, RGB_MOD,
+        _______, _______, _______, _______, _______, _______, _______, _______, BL_TOGG, BL_DEC,  BL_INC,  BL_BRTG
+    )
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
```

Comment:
```suggestion
```

Reply:
Are you sure about this change? I won't be able to access the _ADJUST layer if we remove it

Reply:
VIA has its own implementation of tri-layer, using the `FN_MO13` and `FN_MO23` keycodes and `update_tri_layer()`. The custom `layer_state_set_user()` needs to be removed, otherwise a VIA user wanting to use something other than tri-layer won't be able to configure it.